doc/tokens/authentication_modes.rst

```diff
@@ -24,6 +44,14 @@ separately, as the modes differ in their user experience. For example:
   respective OTP value in the plugin's login form. The plugin sends the
   challenge response to privacyIDEA, which decides whether the authentication
   is valid or not.
+  The "client_mode" is set to ``interactive``". This indicates that
```

Comment:
Please remove the redundant `"` after ``interactive``

Reply:
done in 3326a52e